packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -7238,24 +7238,27 @@ class Semantics extends SingleChildRenderObjectWidget {
 /// the user would not be able to be sure that they were related.
 ///
 /// {@tool snippet}
-/// This snippet shows how to use [MergeSemantics] to merge the semantics of
-/// a [Checkbox] and [Text] widget.
 ///
-/// ```dart
-/// MergeSemantics(
-///   child: Row(
-///     children: <Widget>[
-///       Checkbox(
-///         value: true,
-///         onChanged: (bool value) => null,
-///       ),
-///       const Text("Settings"),
-///     ],
-///   ),
-/// )
-/// ```
+/// here is the sample code to use[MergeSemantics] to merge the semantics of  a [checkbox] and [Text] widget.
+///
+/// ...dart
+/// MergeSemantics{
+///  child: ROW{
+///    children:{
+///     value: true,
+///     onChanged:(value){},
+///    },
+///    Text("Settings"),
+///    ],
+///  ),
+/// ),
+/// ...
+///
```

Comment:
The explanation you added is not clear and has grammatical errors

The code snippet you added is broken, either revert to the original or fix the errors.
Tip: run the code in a flutter project and when everything works just copy and paste into docs.

In your code you use `{}` but you should use `()` and in children `[]` and you removed the checkbox widget and kept only the properties

Reply:
@YazeedAlKhalaf , Thanks for your suggestion. I am working on it .